modules/Planner/planner_view_full_postProcess.php

```diff
@@ -112,7 +115,7 @@
 
                 //Create notification to person I am replying to
                 if (is_null($replyToID) == false) {
-                    $notificationText = sprintf(__('Someone has replied to a comment you made on lesson plan "%1$s".'), $row['name']);
+                    $notificationText = __('{person}  has replied to a comment you made on lesson plan {lessonName}.', ['person' => $personName, 'lessonName' => $row['name']]);
```

Comment:
Looks like an extra space after {person}. Can you also double check that the string in POEditor is correct?
```suggestion
                    $notificationText = __('{person} has replied to a comment you made on lesson plan {lessonName}.', ['person' => $personName, 'lessonName' => $row['name']]);
```